<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View3DTextConfigWidget</class>
 <widget class="QWidget" name="View3DTextConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Font size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinFontSize">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="value">
        <number>12</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Font color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Set font color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShow">
     <property name="text">
      <string>Show labels.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinFontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>View3DTextConfigWidget</receiver>
   <slot>onChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShow</sender>
   <signal>clicked()</signal>
   <receiver>View3DTextConfigWidget</receiver>
   <slot>onChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnColor</sender>
   <signal>clicked()</signal>
   <receiver>View3DTextConfigWidget</receiver>
   <slot>onColorChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChange()</slot>
  <slot>onColorChoose()</slot>
 </slots>
</ui>
